scripts/src/release/releasechecker.py

```diff
@@ -67,6 +67,11 @@ def make_release_body(version, image_name, release_info):
     print(f"[INFO] Release body: {body}")
     print(f"::set-output name=PR_release_body::{body}")
 
+def get_version_info():
+    file = open(VERSION_FILE,)
```

Comment:
comma at the end? I have seen that but prefer you specify the mode explicitly "r" in this case.  one way the I like loading json is this:
data = {}
with open('data.json') as json_file:
    data = json.load(json_file)
return data
This with will handle the opening and closing of the handle.

Reply:
much neater. Used your logic.